opentelemetry-api/tests/metrics/test_globals.py

```diff
@@ -0,0 +1,21 @@
+# type:ignore
+import unittest
+from logging import WARNING
+
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import MeterProvider
+
+
+class TestGlobals(unittest.TestCase):
+    def test_meter_provider_override_warning(self):
+        """metrics.set_meter_provider should throw a warning when overridden"""
+        metrics.set_meter_provider(MeterProvider())
+        with self.assertLogs(level=WARNING) as test:
+            metrics.set_meter_provider(MeterProvider())
+            self.assertEqual(
+                test.output,
```

Comment:
nit: i wouldnt split this string into an array

Reply:
Thanks a lot for the review @codeboten as this is my first PR. If I understand it right you mean 
convert this 
```
                [
                    "WARNING:opentelemetry.metrics:Overriding current "
                    "MeterProvider"
                ]
``` 
to
```
                [
                    (
                        "WARNING:opentelemetry.metrics:Overriding current "
                        "MeterProvider"
                    )
                ]
```
right?

Reply:
I just meant not splitting the expected output over two lines would make it easier to read. Not sure if that would break linting though.

Reply:
Ya it will break the linting. I changed it to use the convention specified in the styling(by enclosing it in curly braces), so the current form should help avoid any confusion.

Reply:
Got it. Thanks